pygbe/output.py

```diff
@@ -26,24 +26,30 @@ def print_summary(surf_array, field_array, param, results_dict):
             rr[ii] = surf_array[i].tree[ii].r
         Levels = int(numpy.log(surf_array[i].tree[0].r/numpy.min(rr))/numpy.log(2) + 1)
 
+        try:
+            key = 'elem_sq_ang_surf{}'.format(i)
+            results_dict[key] = [1/numpy.average(surf_array[i].area)]
+        except IndexError:
+            pass
+
         print('Surface {}:'.format(i))
         print('\t{} elements'.format(N_aux))
-        print('\tSurface type       : {}'.format(surf_array[i].surf_type))
-        print('\tCells              : {}'.format(len(surf_array[i].tree)))
-        print('\tTwigs              : {}'.format(len(surf_array[i].twig)))
-        print('\tLevels             : {}'.format(Levels))
-        print('\tC0 size            : {}'.format(surf_array[i].tree[0].r))
-        print('\tC0 box center      : {}, {}, {}'.format(surf_array[i].tree[0].xc,
+        print('\tSurface type         : {}'.format(surf_array[i].surf_type))
+        print('\tCells                : {}'.format(len(surf_array[i].tree)))
+        print('\tTwigs                : {}'.format(len(surf_array[i].twig)))
+        print('\tLevels               : {}'.format(Levels))
+        print('\tC0 size              : {}'.format(surf_array[i].tree[0].r))
+        print('\tC0 box center        : {}, {}, {}'.format(surf_array[i].tree[0].xc,
                                                          surf_array[i].tree[0].yc,
                                                          surf_array[i].tree[0].zc))
-        print('\tTwig cell size     : {}'.format(numpy.min(rr)))
-        print('\tRbox/theta         : {}'.format(numpy.min(rr)/param.theta))
-        print('\tAnalytic distance  : {}'.format(numpy.average(numpy.sqrt(2*surf_array[i].area))/param.threshold))
-        print('\tElem. per sq Ang   : {}'.format(1/numpy.average(surf_array[i].area)))
-        print('\tMax, min, avg elem.: {}, {}, {}'.format(numpy.max(surf_array[i].area),
+        print('\tTwig cell size       : {}'.format(numpy.min(rr)))
+        print('\tRbox/theta           : {}'.format(numpy.min(rr)/param.theta))
+        print('\tAnalytic distance    : {}'.format(numpy.average(numpy.sqrt(2*surf_array[i].area))/param.threshold))
+        print('\tElem. per Ang^2\n\t(when lspr, per nm^2): {}'.format(1/numpy.average(surf_array[i].area)))
```

Comment:
Can we have a conditional here and print 'per Ang^2' for electrostatics problems and 'per nm^2' for lspr problems?